alerts/cluster-monitoring-operator/PrometheusOperatorRejectedResources.md

```diff
@@ -0,0 +1,93 @@
+# PrometheusOperatorRejectedResources
+
+## Meaning
+
+Prometheus Operator rejected PodMonitor, ServiceMonitor or PrometheusRules objects
+that are misconfigured.
+The alert `PrometheusOperatorRejectedResources` is triggered by misconfiguration
+of these custom resources:
```

Comment:
```suggestion
The `PrometheusOperatorRejectedResources` alert triggers when Prometheus Operator rejects  `Alertmanager` `PodMonitor`, `ServiceMonitor`, or `PrometheusRules` objects because custom resources in an object are not configured correctly:
```